### PR TITLE
Update programme test data to include vls indicators

### PR DESCRIPTION
**Pull request by r-ash**
This adds optional indicators to programme/ART input data
* vls_tested
* vls_suppressed
This also updates metadata so that vls_tested, vls_suppressed and art_new (a column we already had but weren't displaying) can be shown in the input graph

I've deployed this to staging and checked that it all looks OK there. Note that the indicator names not translated yet - that will come when we do the next big set of translations. Which we should do very soon.

### CODE REVIEW

File: docker/build [hunk 1]
```diff
@@ -29,7 +29,7 @@ ENTRYPOINT [ "/bin/bash", "-l", "-c" ]
 EOF
 
 rm -rf naomi
-git clone https://github.com/mrc-ide/naomi
+git clone --single-branch --branch simulated-vls-data https://github.com/mrc-ide/naomi
```
Comment (richfitz) [inline docker/build:32]: Should this PR stay open until this is merged?

**Review by richfitz — COMMENTED** [review]
A classic of the genre of "nothing wrong with the code, does the data to the right thing for the front end though?"

Looks like there's a naomi PR that needs merging (https://github.com/mrc-ide/naomi/pull/258) before this can go in

**Review by EmmaLRussell — COMMENTED** [review]
Tested this locally with Antarctica dataset from ADR, and both indicators are shown in drop-down, but neither have any data available (it's not returned in the programme data). Do we have any input files which should result in data for these indicators?

Screenshot from 2021-11-09 14-10-22
Screenshot from 2021-11-09 14-10-37

**Comment by EmmaLRussell** [comment]
> Tested this locally with Antarctica dataset from ADR, and both indicators are shown in drop-down, but neither have any data available (it's not returned in the programme data). Do we have any input files which should result in data for these indicators?

Actually, maybe I was using the wrong Antarctica..

**Comment by jeffeaton** [comment]
We probably have not uploaded the testing datasets with VLS to ADR yet.  We should be fine to do that though.

From: EmmaLRussell ***@***.***>
Date: Tuesday, 9 November 2021 at 14:46
To: mrc-ide/hintr ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [mrc-ide/hintr] Update programme test data to include vls indicators (PR #310)

Tested this locally with Antarctica dataset from ADR, and both indicators are shown in drop-down, but neither have any data available (it's not returned in the programme data). Do we have any input files which should result in data for these indicators?

Actually, maybe I was using the wrong Antarctica..

**Comment by r-ash** [comment]
I've updated the Naomi development team version of the ART data to include these new columns. Also a local version of test data available in this PR

**Review by EmmaLRussell — APPROVED** [review]
Looks good, data now showing in the map and table

**Pull request merged**